### PR TITLE
Update README.md with troubleshooting reference

### PR DESCRIPTION
**Pull request by miguelvr**
Adds reference to the troubleshooting document in the main README.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -204,6 +204,11 @@ the maintainers of the project found in the [MAINTAINERS.md](MAINTAINERS.md) fil
 
 [operator-vs-kube]: https://github.com/prometheus-operator/prometheus-operator/issues/2510#issuecomment-476692399
 
+## Troubleshooting
+
+Check the [troubleshooting documentation](Documentation/troubleshooting.md) for 
+common troubleshooting issues and frequently asked questions (FAQ).
```
Comment (lilic) [inline README.md:210]:
```suggestion
common issues and frequently asked questions (FAQ).
```

**Review by lilic — COMMENTED** [review]
Just one small suggestion. Otherwise lgtm, thank you! :)

**Comment by miguelvr** [comment]
done 😄

**Comment by lilic** [comment]
Thanks, lets wait until CI is done, and we can merge this 🎉

**Pull request merged**